### PR TITLE
Added solution to HW2-2

### PR DESCRIPTION
**Pull request by MichaelShevchenko**

### CODE REVIEW

File: src/main/java/core/basesyntax/MainApp.java [hunk 1]
```diff
@@ -12,5 +12,16 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine track1 = new Track();
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/MainApp.java:16]:
```suggestion
        Machine track = new Track();
```
bad variables naming

File: src/main/java/core/basesyntax/MainApp.java [hunk 2]
```diff
@@ -12,5 +12,16 @@
...
+        Machine bulldozer1 = new Bulldozer();
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/MainApp.java:17]: bad variables naming

File: src/main/java/core/basesyntax/MainApp.java [hunk 3]
```diff
@@ -12,5 +12,16 @@
...
+        Machine excavator1 = new Excavator();
+        Machine[] objectsArray = {track1, bulldozer1, excavator1};
```
Comment (boroda4436) [inline src/main/java/core/basesyntax/MainApp.java:19]: bad variables naming

File: src/main/java/core/basesyntax/MainApp.java (hunk 1 above)
Comment (MichaelShevchenko) [reply src/main/java/core/basesyntax/MainApp.java]: done

File: src/main/java/core/basesyntax/MainApp.java (hunk 2 above)
Comment (MichaelShevchenko) [reply src/main/java/core/basesyntax/MainApp.java]: done

File: src/main/java/core/basesyntax/MainApp.java (hunk 3 above)
Comment (MichaelShevchenko) [reply src/main/java/core/basesyntax/MainApp.java]: done

**Pull request closed without merging**